src/Build/BackEnd/Components/Logging/ILoggingService.cs

```diff
@@ -477,15 +486,26 @@ MessageImportance MinimumRequiredMessageImportance
         /// </summary>
         /// <param name="nodeBuildEventContext">The logging context of the node which is building this project.</param>
         /// <param name="submissionId">The id of the build submission.</param>
-        /// <param name="projectId">The id of the project instance which is about to start</param>
+        /// <param name="configurationId">The id of the project configuration which is about to start</param>
```

Comment:
Renamed this for clarity. Personally, I understand `configurationId` as a concept more than `projectId`, although admittedly the `ProjectId` is exposed on a public API while internally it's pretty much always `configurationId`. So I'm open to reverting the rename if desired, but my reasoning is that this is an "internal" name here so should be consistent with other internal usage.